main.py

```diff
@@ -22,7 +22,10 @@ def profile():
 # log in required to use search feature???
 @main.route('/search')
 @login_required
-def search():
+def search(): 
+    form_data = request.args
+    query = form_data.get("searchTerm")
+    # print(query) # Printing out the search term that user puts in the search bar to confirm if the search word is being sent to the main.py.
```

Comment:
1) I'm glad you tested it and commented out this line after trying to see if it works, but can you please remove it and recommit it?
2) Can you also please leave some comments for future reference as to where this term is coming from and give it more context, just in case we want to come back in the future and see what's going on? Thank you!

Reply:
I removed the commented line and added more comments to explain the code.